examples/license-classifications.yml

```diff
@@ -5,374 +5,375 @@
 # To demonstrate how one can insert a custom written offer
 # include_source_code_offer_in_notice_file has been set to
```

Comment:
commit-message: The idea behind having a `name` and `id` was to have a property for a human readable name and a property which is uniquely identifiable. Identifiers could be maybe even follow a scheme or be defacto standardized accross companies. 

I'm completely fine with dropping one of the two, but I think we should keep the `ìd` and drop the `name` as `identifiers` are `unique` which is required, and as names may be subject to localization which is not what is needed here.

Reply:
The reasoning for dropping `id` was that the field is a free, human-readable String, in contrast to a typically more technical ID; but you have good arguments for setting on `id` rather than `name`. I would be fine with both terms.

Regarding `classifications`: I was not sure about the term either, at least it fits to the name of the file. I am open for suggestions.

Reply:
@sschuberth @mnonnenmacher  considering above comments, do you have any preference?

Reply:
> names may be subject to localization

I didn't think of it that way, but more like "company name" or "personal name", which are not localized. `id` sounded to me too much like something machine readable that is not speaking, but in fact I believe that categories would have semantics implied, which is why I found a more human-friendly `name` to be more fitting here. Also, it nicely separates from `classification -> id`, which this time *really* is an `id` in my view.

> naming it `licenses` IMO is not that unclear as that array contains a list of `License` instances.

I'm against keeping this field as "licenses" as it can be too easily misinterpreted as *the* (complete) list of licenses ORT supports / is able to recognize, which is not true. This is just the sub-set of licenses for which a classification is done.

Regarding the "classifications" field: Thinking about is again I believe to have a slight preference for "categorization(s)" now. That would fit the "categories" field, and it would also fit "classifications" from the file name as the general topic, as [the process of arranging things by category *is* a classification](https://wikidiff.com/classification/categorization).

Reply:
I like "categorizations". @fviernau, would this be okay for you as well?

Then we still have to agree on either "id" or "name" for categories.

Reply:
@oheger-bosch I'm fine with 'categorizations'. Regarding 'name' vs 'id' I'm fine with both choices.